### PR TITLE
X x adr xx patch 1

### PR DESCRIPTION
**Pull request by xXAdrXx**
Made the neccessary changes

### CODE REVIEW

File: _posts/2024-03-21-fixing-a-godot-doc.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Godot is a free and open source game engine available for free download from [the Godot website](https://godotengine.org/).
+
+## Why this project?
+I recently did a Comm Arch about godot and how easy it is to make edits, so I thought it would be something nice to contribute to
+
+## Comm Arch Experience
+My Comm Arch experience told me that this would be a good project to contribute to. Godot is heavily documented as is its contribution process & onboarding
+The Area Owners/teams can assist as well with questions about certain sections. They even had instructions for making changes to anything such as page editing documentation(which is shown below)
+
+![page editing documentation edit](assets/2024-04-18-Adrian_Marquez_godot_docfix/page_edit_documentation.png)
```
Comment (MoralCode) [inline _posts/2024-03-21-fixing-a-godot-doc.md:27]: seems like the images still arent loading, try this: 
```suggestion
![page editing documentation edit](../assets/2024-04-18-Adrian_Marquez_godot_docfix/page_edit_documentation.png)
```

File: _posts/2024-03-21-fixing-a-godot-doc.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## The Issue
+I choose my Issue by starting very small. I viewed the 'good-first-issue' tag.I had no experince with godot code or coding within godot and 
+so when looking through the issues I found a bug that was centered around a  built-in CAMERA_VISIBLE_LAYERS Shader being undocumented according to the issue, this Shader was added back in 2022 but nobody had documented yet. I felt like that was something I could fix and so I went right to it
+
+![image of the issue](/hfoss2024-blogs/posts/assets/2024-04-18-Adrian_Marquez_godot_docfix/Image_of_issue.png)
```
Comment (MoralCode) [inline _posts/2024-03-21-fixing-a-godot-doc.md:33]:
```suggestion
![image of the issue](../assets/2024-04-18-Adrian_Marquez_godot_docfix/Image_of_issue.png)
```

try applying this to the rest of the images

**Review by MoralCode — CHANGES_REQUESTED** [review]
Looking better! Just a quick fix to the image paths since they still dont seem to be working.

Also, for future reference, if you push commits to the branch you have a Pull Request for, it will automatically add those commits to the pull request, you dont have to create a new PR every time.

hope this helps!

**Comment by xXAdrXx** [comment]
I think I made the necessary changes this time

File: _posts/2024-03-21-fixing-a-godot-doc.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## The Issue
+I choose my Issue by starting very small. I viewed the 'good-first-issue' tag.I had no experince with godot code or coding within godot and 
+so when looking through the issues I found a bug that was centered around a  built-in CAMERA_VISIBLE_LAYERS Shader being undocumented according to the issue, this Shader was added back in 2022 but nobody had documented yet. I felt like that was something I could fix and so I went right to it
+
+![image of the issue](../assets/2024-04-18-Adrian_Marquez_godot_docfix/Image_of_issue.png)
+## Was I successful?
+The fix was simple especially with the guidence they give to people who want to fix or add any documentation and so I went into the appropriate file within the repository and added the necessary change
+
+![image of the fix](/hfoss2024-blogs/posts/assets/2024-04-18-Adrian_Marquez_godot_docfix/Image_of_fix.png)
```
[the remark this answers is not in the document]
Comment (xXAdrXx) [reply _posts/2024-03-21-fixing-a-godot-doc.md]: i think i did it right this time

[the remark this answers is not in the document]
Comment (MoralCode) [reply _posts/2024-03-21-fixing-a-godot-doc.md]: @xXAdrXx very close! i think in your fix for this particular image you forgot to remove the `posts` part of the URL like you did for the rest of them

**Comment by xXAdrXx** [comment]
please tell me i did it right

**Comment by MoralCode** [comment]
> please tell me i did it right

Almost! take a look at the comment I left in the review suggestion above - seems like you forgot to fix one image path

**Comment by MoralCode** [comment]
Nevermind i didnt see the commit you added. LGTM!

**Pull request merged**